plugins/ci/skills/analyze-disruption/download_timelines.py

```
@@ -0,0 +1,185 @@
+            sys.exit(1)
+        job, build_id = pair.rsplit(":", 1)
+        if not job or not build_id:
+            print("Error: empty job or build_id in %r" % pair, file=sys.stderr)
+            sys.exit(1)
+        runs.append((job, build_id))
+    return runs
+
+
+def download_file(gcs_path, local_path):
+    """Download a single file from GCS. Returns True on success."""
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)